notebooks/en/structured_generation_vision_language_models.ipynb

```diff
@@ -0,0 +1,402 @@
+{
```

Comment:
<p>Looks like there are still some typos here!</p><p> </p><p>"...from documents. We will run the VLM using the Hugging Face Transformers..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/279/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>